spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.